string[] objects
DetectedObject[] ob_msgs